### ISSUE TITLE
How can I configurate formatting of ternary operator?

### ISSUE THREAD

**Issue by doox911** [dprint/dprint-plugin-typescript#432]
For example:

```
const value = is_prod 
  ? do() 
  :  is_laptop 
    ? do2()
    : do3();
```

Language: Javascript, Typescript, 
Files: Vue, React, jsx and tsx.

**Comment by dsherret** [comment]
Sorry for my delay. Do you mean the operator position? You can use the `"conditionalExpression.operatorPosition"` configuration for that (see https://dprint.dev/plugins/typescript/config/ and search for "operatorPosition").

Or are you wondering how to indent the second conditional expression? It's not possible to do that at the moment.

**Comment by doox911** [comment]
Yes, it is nesting(second conditional expression) that interests me. Then let it be a wish for improvement)!

**Comment by unicornware** [comment]
@dsherret what option can i use to maintain the formatting below?

```ts
/**
 * @file Type Definitions - Writable
 * @module tutils/types/Writable
 */

/**
 * Construct a type where all properties of `T` are writable.
 *
 * This is the opposite of {@linkcode Readonly}.
 *
 * @todo examples
 *
 * @template T - Type to evaluate
 */
type Writable<T> = T extends unknown
  ? { -readonly [K in keyof T]: T[K] }
  : never

export type { Writable as default }
```

i tried changing `operatorPosition` (and its variants), but the [new formatting][1] is still ugly.

[1]: https://dprint.dev/playground/#code/PQKhCgAIUgBAzAlgGwKaQCoE8AO6AiqSAdogC6ID2xAzpALSQDqATuQIYBGaUMsAtpQAmAVzSQyIishrAyuVLNYduqXsHDhQEaJADC1GmRYiAxmUjsJCyAHcAFqhbp2yZJBwtKeFhUWRKeEgAAwxgy2c7NjIuNAA6Xl5Me0Q6VIlHAJwcShpydEDIAG9YZERiAGtTYXQAJVR2IWpkLABfBOgk2DJhSkhUAA92fhw0GkTdbtQR5HYydAwGTBse-oA3VxE5tWgNeTxmaNjUAB4MAD5IAF5MfoH54iE6EUriSltiKEgAfmKGZ0azSwkAA2gBpSDlSAVVBYQoYAC6AC5MOCEZBWl8UcRUGsnJpBjlfNYDkVDipxOw6EIiOwxBZMUA/config/N4KABBBEDGD2B2ATAlgF2QghgGwCoE8AHAUwDpYSAnTVWSgBVgGc0N5IAuMSAW02Xip+7ADTgI3TJQDmAVx7FBTUoUrEAZsUoAJTPGkDpnbvGIA3LZDETJlavgCiAD1XEmLBCrWadeg-uNIUwtKK3EoKUpYAHcAMVl4aHRPWSZieilFVAALNzdA1x8AOQRiMJtIACMBKUcXNXc2UmwBdK1nV0aEY3UcNOsJKprKOs6PeHIqGjpGFmT2LkgmTAUAGVbywer4Wo6G8dImQkxoYgBlWTtYBJR9ACE0aOQ0gEEkF8o0bIV0aAB5Ka0UJcVCUWTEAZQSrUU6zVjdRbLNYbSHcOA8BSCdamUjqOinM7HU4vdSoLRnbCYJi5JjGUHg1EwBBMelJM7IaQ7VCXMhHE7EElkyhFYjRADSxHw0ToiDpYIh4TRzNZQMOROIdw0dHSmUENPyXF62H6iqZ8BZYKSdAIJDV-MFWhF4sl0sospB8sZiFgAHVsshsOchGTMag7cTSVo-QHiBKpTK5QzTYp5AARYjQSnUeakBQ8SrkomGQJ8ARCASbKDEFx0VDpzNSGhNPGUU4AWVk2HQ2LKhr6Coq1cItfrWabnhbBMMgZ7PX7jKHI4zY5zTFrRRWxEQHVrtMWpcEwkr3EXlDry8bq-VFyuN0MG4U25rZ73YHpA8GLYAkvAzsHiKG4YCpGlCxHQcauu6b6eqaLZ-Oof40ABWRAQ6oHgS6CYekmFQtohIYoXyEZCmBlAQVh0E4Z+dD4chgioSBZwAcgADCsDYMyiYflA6gJEkbCjpeTRERqWpqBkah6nkr5GiauF8fMexuAcIlofEiTzORbpzsa3HcLxGlsEpXQTCJmotjqkk5NJOlyYM0jEKgLzQKc7h0EB5nahJWT6jJ86msgCH-oBqkgV+6haVB76Mhy8DarEMZsRiWS4NWqCBIgqhlgAtLF2rZeoMbHpAeVqCUiDEEloapU46WLJlnyCLlnLasVyA8MOZ6Cdmzb4sQHZdsgs59rpMUdUuDY9ROfXsvoM4bCNdlQO1nXnpN46meum6IF+40viWwjlqIAV7WtK7CdelxRHe+gPluu2ra+0WmgAVkwTikNCJwANaOXC8yBKYtWzoyb0fZOxBOj2TA3tdSCGGxghZLZemQGDuadt2rTeeagQ4NEmD4LSoPvaQACOsiwGSf74IGBTeOS05lCTTjGeMiNHVoQGw9c8P6GzbBcSzTHYOoLEcSw-OBiF6qeWoFJUtkQumnmBaUALngtKY9DtPUymC4tqM-NksCIB5YmWT5NmG4yPCm52vK1juXUXlNuOLNAVLnIoczIBYxV24gDuHLWD0TednEe17THmqw-uMqY0Q9mKAhQZAovFUDqAc1E2CxBx0T-Qb3BIsQIOmrAlQvRmqAa6Zl23nz0j0FEVDoAalGo5X1dJBkqBCvX-I8zdzet1o7dPTBFQUFo0wMMw8ILCYaXlxUxzUD8WjKIUWi6PoxaLMEliMjvvj7wE2Go6fs3SPNpgo4yFNU+krevpAVIiluW7FU-1OoLTvZJDYAJkTG+dNGRpB4KxdikdJAsGKmkJyLllLuTMhbHG1k0h+VGqaSWt8NSm3wEXBEJdNyr0GKpOGUFZKoxEsPJuLcZ5nmQB3Z6FQhDSAcogVKHVKRknNhZdYQocAP1NKCfgWtpBVSpIDcwx8xGyEIIGG0ZBT5738EYQ+cjQiMn-iQN4cVgwXX5HLSqMCl40N0UQAU7hx7GNOKYuuoj2HWKEbPbAhxiDrzninJAhwmbDRLsxNiHFjouJIG46gHi6FXV5rcUeTCJ7K3CZZTcQpt4MzPho2RIRiqpA1DCDuvBDpHkZNEf0gZaIy3tCBaMgZIpcRAAAXxAEAA/language/typescript

**Comment by germangorodnev** [comment]
Would be very good to have this merged!

**Comment by mikethea1** [comment]
It would be great for us to support switch-style ternary formatting:

```
const value = condition1 ? value1
    : condition2 ? value2
    : condition3 ? value3
    : value4;
```
